modules/default-requirements-central-services.adoc

```diff
@@ -32,12 +32,11 @@ recommend using a hostPath volume.
 ====
 * Use Solid-State Drives (SSD) for best performance.
 However, you can use another storage type if you do not have SSDs available.
-* If you use a web proxy or firewall, you must configure bypass rules to allow traffic for the `definitions.stackrox.io` and `collector-modules.stackrox.io` domains and enable {product-title} to trust your web proxy or firewall. Otherwise, updates for vulnerability definitions and kernel support packages will fail.
+* If you use a web proxy or firewall, you must configure bypass rules to allow traffic for the `definitions.stackrox.io` domain and enable {product-title} to trust your web proxy or firewall. Otherwise, updates for vulnerability definitions will fail.
```

Comment:
Do we need to add "For version 4.4 and earlier, you might need to enable the `collector-modules.stackrox.io` domain." ?